subprojects/core/src/main/java/hu/bme/mit/theta/core/dsl/impl/StmtWriter.java

```diff
@@ -15,11 +15,7 @@
  */
 package hu.bme.mit.theta.core.dsl.impl;
 
-import hu.bme.mit.theta.core.stmt.AssignStmt;
-import hu.bme.mit.theta.core.stmt.AssumeStmt;
-import hu.bme.mit.theta.core.stmt.HavocStmt;
-import hu.bme.mit.theta.core.stmt.SkipStmt;
-import hu.bme.mit.theta.core.stmt.StmtVisitor;
+import hu.bme.mit.theta.core.stmt.*;
```

Comment:
We prefer not to use `import *`. I will not add this comment, but should be fixed in other places.